core/src/com/mygdx/game/client/screen/GameScreen.java

```diff
@@ -58,15 +68,19 @@ private void positionCamera(@NonNull Camera camera) {
   public void show() {
     positionCamera(viewport.getCamera());
 
+    stage = new Stage(new ScreenViewport());
```

Comment:
wydaje mi się, że ten screenviewport sprawi, że później reszta UI będzie się rozwalać i nie widać tego na razie z dialogiem, bo on sam siebie pozycjonuje i ustawia wielkość

ale na razie ok, później się zobaczy jak będą dodawane inne UI,

nie wiem czy instacjonowanie Stage jest całkowicie ok (bo myślałem, że to jakoś przez daggera będzie robione), ale może to nic